test/testing_test.rb

```diff
@@ -213,11 +213,32 @@ def plan_input
         end
       end
 
-      it "is able to handle when events ===" do
+      it "is able to handle when events" do
```

Comment:
How is this test different from [this one](https://github.com/iNecas/dynflow/blob/21173d948e3fc40fc5d05015595240fc738de656/test/testing_test.rb#L204)?